.circleci/config.yml

```diff
@@ -29,6 +29,16 @@ jobs:
       - store_artifacts:
           path: ui-test-release.html
 workflows:
-  build_and_test:
+  commit_workflow:
     jobs:
-        - test
+      - test
+  stage_release_run:
```

Comment:
Couuld we add some context to this title showing that it is a scheduled job?

Reply:
So, basically, I should move the `# the workflow runs each Wednesday at 05:00 UTC` comment here?

Reply:
Not quite, maybe change it to like ```scheduled_stage_release_run```